### DISCUSSION TITLE
🎃. Welcome Hacktoberfest 2021!

### DISCUSSION THREAD

**Discussion by keithrfung** [microsoft/electionguard#123] [General]
We love the open source community. If you're coming from [Hacktoberfest](https://hacktoberfest.digitalocean.com/) and have questions. Feel free to ask. Drop a message in the discussion and say Hi!

![](https://media4.giphy.com/media/5knp3kuvvvnW/200.gif)

**Lost? Check out this [page!](https://www.electionguard.vote/hacktoberfest/)**

**Comment by gdeep-ciena** [comment]
@keithrfung  Can you please add more hacktoberfest issues to electionguard-ui repo

**Reply by keithrfung** [reply]
We're still really busy on setting up that repository, but I'll do what I can.

**Reply by keithrfung** [reply]
I'm slowly working on adding some. Expect more.

**Comment by amarealcoder** [comment]
I would like to work on documentation if its needed. Can I get direction where to start?

**Reply by keithrfung** [reply]
@amarealcoder This issue in the c++ repository might be up your alley. 
https://github.com/microsoft/electionguard-cpp/issues/18

We will be posting new issues monday as well.

**Reply by keithrfung** [reply]
@amarealcoder I added a ton of new documentation cards. Feel free to take a swing.

**Comment by nathannunes** [comment]
where can i begin?

**Reply by keithrfung** [reply]
@nathannunes What is your code background? Have any specialties?

The python repository is usually people's first starting point. It's linked here for fast finding.
https://electionguard.vote/hacktoberfest

**Reply by nathannunes** [reply]
Hi, I work with Java, but am okay with Js, Typescript as well. Python is okay too

**Reply by keithrfung** [reply]
I'd definitely recommend starting with electionguard-python or electionguard-ui (though there are minimal issues as of right now)

**Reply by nathannunes** [reply]
Thanks Keith I'll get started with those, maybe I'll explore c++ later

**Reply by keithrfung** [reply]
Awesome. Glad to have you diving in. Feel free to reach back here.

**Reply by keithrfung** [reply]
A bunch of new documentation cards were added if you wanted any more places to start.

**Comment by anishsingh42** [comment]
Hello ,
How can I contribute under hacktoberfest-2021 under this repo.
Tech Stack I am familiar with :C++(Basic-Intermediate level), HTML and CSS3(Basic level)

**Reply by keithrfung** [reply]
Welcome! If you interested in C++, I'd highly recommend our [c++ library](https://github.com/microsoft/electionguard-cpp)

**Comment by Anubhab2002** [comment]
Hi, I am a beginner with experience in python(flask included), HTML, CSS(Bootstrap), Javascript(React, React Native, Node) mainly. Can work with C++ as well. How do I contribute to this project under Hacktoberfest 2021?

**Reply by keithrfung** [reply]
@Anubhab2002 

If you navigate to our contribute page, you'll see many of our repositories. It sounds like electionguard-python , electionguard-api-python and electionguard-ui would be good fits. electionguard-cpp maybe as well. 

electionguard.vote/contribute

Looks for `hacktoberfest` labeled issues and start with those. If you can't find any, reach back here and I'll see what I can do.

**Comment by amarealcoder** [comment]
Alright Keith, I will check them out.
